docs/components/MessagingAttachmentView.jsx

```diff
@@ -0,0 +1,264 @@
+import { FileAttachmentIcon } from "src/MessagingAttachment/MessagingAttachment";
+
+import "./MessagingAttachmentView.less";
+
+const cssClass = {
+  CONFIG_CONTAINER: "MessagingAttachmentView--configContainer",
+  CONFIG_OPTIONS: "MessagingAttachmentView--configOptions",
+  CONFIG: "MessagingAttachmentView--config",
+  CONFIG_LABEL_TEXT: "TextInput2View--configLabelText",
+  CONFIG_TOGGLE: "MessagingAttachmentView--configToggle",
+  CONTAINER: "MessagingAttachmentView",
+  INTRO: "MessagingAttachmentView--intro",
+  PROPS: "MessagingAttachmentView--props",
+  DEMO_ROW: "MessagingAttachmentView-demoRow",
+};
+
+// TODO: FINISH THIS DOCUMENT
+export default class MessagingAttachmentView extends React.PureComponent {
+  static cssClass = cssClass;
+
+  render() {
+    const sentAttachments = [
+      {
+        attachmentID: "1",
+        icon: <FileAttachmentIcon fileType={"doc"} />,
+        onClickAttachment: () => console.log("clicked!"),
+        title: "MyCoolDoclajsdjasldjaslkdjasldkjasldjaslkdjasldjasldjalskjdalskjdaslkjasljd.doc",
+        subtitle: "Click to download",
+      },
+      {
+        attachmentID: "1",
+        icon: <FileAttachmentIcon fileType={"image"} />,
+        onClickAttachment: () => console.log("clicked!"),
+        title: "Flyer.png",
+        subtitle: "Click to view",
+      },
+      {
+        attachmentID: "1",
+        icon: <FileAttachmentIcon fileType={"audio"} />,
+        onClickAttachment: () => console.log("clicked!"),
+        title: "Morning message.m4a",
+        subtitle: "Click to download",
+      },
+    ];
+
+    const draftAttachments = [
+      {
+        attachmentID: "1",
+        icon: <FileAttachmentIcon fileType={"doc"} isUpload />,
+        onClickAttachment: () => console.log("clicked!"),
+        onRemoveAttachment: () => console.log("removed!"),
+        title: "MyCoolDoclkasjdlkajsdlkasjdlkasjdlkasjdlsakjd.doc",
+        subtitle: "Uploading 5%",
+        isUpload: true,
+      },
+      {
+        attachmentID: "2",
+        icon: <FileAttachmentIcon fileType={"image"} isUpload />,
+        onClickAttachment: () => console.log("clicked!"),
+        onRemoveAttachment: () => console.log("removed!"),
+        title: "Flyer.png",
+        subtitle: "Processing...",
+        isUpload: true,
+      },
+      {
+        attachmentID: "3",
+        icon: <FileAttachmentIcon fileType={"audio"} isUpload uploadComplete />,
+        onClickAttachment: () => console.log("clicked!"),
+        onRemoveAttachment: () => console.log("removed!"),
+        title: "Hover over me.m4a",
+        subtitle: "Click to download",
+        isUpload: true,
+        uploadComplete: true,
+      },
+      {
+        attachmentID: "4",
+        icon: <FileAttachmentIcon fileType={"audio"} isUpload uploadError />,
+        onClickAttachment: () => console.log("clicked!"),
+        onRemoveAttachment: () => console.log("removed!"),
+        title: "File cannot exceed 10 MB",
```

Comment:
Shouldn't this be `errorMsg` instead of `title`?

Reply:
Genius